docs/GPS.md

```diff
@@ -1,29 +1,37 @@
 #Phidget GPS Sensor
 
-The PhidgetGPS library makes for intuitive and lightning fast development without any compromise. For a quick start into your GPS project see the __Getting Started__ section.
+The PhidgetGPS library makes for intuitive and lightning fast development without any compromise. For a quick start into your GPS project see the [Basic Manager example](https://github.com/RIAEvangelist/node-phidget-API/blob/master/examples/GPS.js).
```

Comment:
all of the changes in this file should have been done via a [git fetch upstream](https://help.github.com/articles/syncing-a-fork/) looks like they are OK this time, as I don't see any conflicts.